### PR TITLE
Tammy Fisher Sprint Challenge 1

### PR DESCRIPTION
**Pull request by TammyLFisher**
Hi, I have completed the sprint challenge. I would like to work on the stretch goals, however, I will try to complete them this weekend. It's late here, so as I work on these projects, it gets deep into the night, I'm in  CST time zone, Please let me know how I did on this project. Thank You, 
Tammy Fisher

### CODE REVIEW

**Comment by TammyLFisher** [comment]
Hi, I finally got the Sprint Challenge Stretch goals done. I had a difficult time getting my animations to work for me, so I am hoping they work for you. I also had a hard time getting the contact page to work properly, but now it leads back to the home page. The other problem I had was getting my media queries to work for the contact page, I haven't figured out how to test it, but hopefully it works. 
Thank you.

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                    <a href="#">Blog</a>
+                    <a href="contact.html">Contact</a>
+                </nav>
+        </header>
+        <img class="blog-image" src="https://i.imgur.com/mqkHY0G.jpg">
+        <div class="blog-content">
+            <section class="top-content1">
+                <h4>Lambda Blog</h4>
+                <p class="introduction">This blog was created to discuss the learning experiences of all of Lambda
+                    school's cohorts and their
```
Comment (liam-mei) [inline blog.html]: I love the blog!  It's so creative!  It's cool how the hover flips the text in different directions.

File: css/index.css [hunk 2]
```diff
@@ -143,4 +218,255 @@ footer nav {
...
+	background: white;
+	border: 1px black solid;
+	cursor: pointer;
+	margin-top: 10px;
+}
+.paragraphs button:hover {
+	background: crimson;
+	color: white;
+}
+.blog-content{
```
Comment (liam-mei) [inline css/index.css]: If anything, it might be a little cleaner if the background image was slightly opaque so that it didn't have such a large contrast

Comment (liam-mei) [reply css/index.css]: https://guide.freecodecamp.org/css/background-opacity/

**Comment by liam-mei** [comment]
Tammy Fisher

**Self Study Questions**
complete

**MVP Objectives**
## Home Page
- [x] Build the HTML and CSS to create the missing navigation and header.
- [x] Link the About navigation item to the about.html page
- [x] Color The Boxes Correctly

## About Page
- [x] Copy and paste your home page navigation and header into the about page
- [x] Update the header image with the about page image
- [x] Link the Home navigation item back to the index.html page.
- [x] Build the rest of the about page layout to match the design
--------------------------------------------------------------------------

**Additional Comments**
- [x] Follows Lambda Git Flow
- [x] Frequent, clear, and descriptive commits
- [x] Code is organized and easy to read
- [ ] Code follows consistent naming conventions, language idioms, and style.
    some of your CSS still have the 1 like text-container1 even though there is only 1 or your p class.
- [x] code is DRY (Don't Repeat Yourself)

--------------------------------------------------------------------------
**Stretch**

- [x] Build a page of your choosing from the navigation items. Come up with content and images that fit the theme.
- [x] Introduce CSS animations to your site.
- [ ] Build a contact page and create a form with several inputs of your choosing
- [ ] Add responsive breakpoints to your code by using media queries

--------------------------------------------------------------------------
Score: 3

--------------------------------------------------------------------------

Great job with the colored boxes!  I think your implementation is great!  You rewrote the 3 nested classes to just 1 class and reused those classes to assign the color as well.

Great Commits  nice and descriptive!  I can see what you did!!!  Commits are great!

I see you added the DIVs for the boxes.  It looks to me as if you're making great strides on how to view a page and know how you want to structure it!!!

I love the background and animations!

**Pull request merged**